pkg/handler3/handler_test.go

```diff
@@ -147,39 +147,39 @@ func TestRegisterOpenAPIVersionedService(t *testing.T) {
 			respBody:     returnedJSON,
 			expectedETag: computeETag(returnedJSON),
 		}, {
-			acceptHeader: "application/com.github.proto-openapi.spec.v3@v1.0+protobuf",
+			acceptHeader: "application/com.github.proto-openapi.spec.v3.v1.0+protobuf",
```

Comment:
I think we should also test the returned `ContentType` in this test-table. Make sure that we return the same mime-type for both deprecated and non-deprecated protobuf headers.

Reply:
@apelisse   I have returned content type and added tests to verify that corrected content type is returned with deprecated content-type header in request.